### DISCUSSION TITLE
[Solved] Hide sitemap & 404 on Expo Router Tabs

### DISCUSSION THREAD

**Discussion by aliansyahFirdaus** [nandorojo/solito#334] [Q&A] [answered]
I initiated an expo project using the expo router template, when I want to create a tab in _layout.tsx, a sitemap tab appears and a [...404] tab, how do I hide this?

<img width="411" alt="Screenshot 2023-02-22 at 12 32 47" src="https://user-images.githubusercontent.com/74974463/220532149-701fe7cb-227c-4e65-a921-4616eb8b9484.png">

**Comment by nandorojo** [comment, marked as answer]
I'd reference the expo router docs.
